src/crypto.c

```diff
@@ -287,7 +286,6 @@ bool crypto_get_compressed_pubkey_at_path(const uint32_t bip32_path[],
     bool result = true;
     BEGIN_TRY {
         TRY {
-            keydata.prefix = 0x04;  // uncompressed public keys always start with 04
```

Comment:
Do we not need this anymore ?

Reply:
The memcpy below now copies the entire 65-byte uncompressed pubkey before (including the prefix), so the code is simpler. Earlier, the first byte was skipped is the memcpy, so I was adding the prefix by hand. Uncompressed pubkeys always start with `0x04`.
The change is to make the code analyzers happy, as casting a `struct` to a `uint8_t *` made them _nervous_.